integration-tests/src/test/java/com/datastax/oss/driver/mapper/StatementAttributesIT.java

```diff
@@ -198,11 +243,23 @@ private void validateQueryOptions(QueryLog log) {
     @Insert
     void save(Simple simple, StatementAttributes attributes);
 
+    @Insert
+    @Attributes(consistencyLevel = "ANY", serialConsistencyLevel = "QUORUM", pageSize = 13)
```

Comment:
This is probably my OCD speaking, but it bothers me that the annotation does not use the same terminology as the parameter type. Of course it would be ludicrous to name them the same, but it would be nice to have some consistency.
My suggestion is `@WithStatementAttributes`, but I'm open to other ideas.